client/asset/dcr/dcr.go

```diff
@@ -53,7 +53,7 @@ const (
 )
 
 var (
-	requiredWalletVersion = dex.Semver{Major: 8, Minor: 3, Patch: 0}
+	requiredWalletVersion = dex.Semver{Major: 8, Minor: 4, Patch: 0}
```

Comment:
To test this PR against dcrwallet master, please undo this change. Making the change now so "we" don't forget to bump this version requirement before merging.